tests/generators/test_rest.py

```diff
@@ -95,3 +95,29 @@ def test_json_rest_deeper(requests_mock):
     generator = RestGenerator()
     output = generator._call_model("Who is Enabran Tain's son?")
     assert output == [DEFAULT_TEXT_RESPONSE]
+
+
+@pytest.mark.usefixtures("set_rest_config")
+def test_rest_skip_code(requests_mock):
+    generator = _plugins.load_plugin(
+        "generators.rest.RestGenerator", config_root=_config
+    )
+    generator.skip_codes = {200}
```

Comment:
Not required but would prefer this be a list for same reason default is a list type.
```suggestion
    generator.skip_codes = [200]
```